src/app/modules/consultations/consultation-profile/consultation-response-text/consultation-response-text.component.html

```diff
@@ -1,5 +1,56 @@
-<div class="response-index mt-4" ngResizable [rzHandles]="'s'"
-#responseIndex>
+<div class="paper-effect  mt-4 m-0-sm">
```

Comment:
@Nishanthh Bro is it possible to make this as a component since its being used both in consultation-response-text.component.html  and consultation-questionnaire.component.html?

Reply:
@avikt yes bro, make into seperate component